src/dataModels/JSON.js

```diff
@@ -579,37 +548,9 @@ var JSON = DataModel.extend('dataModels.JSON', {
 
     /**
      * @memberOf dataModels.JSON.prototype
-     * @param {number} colIndex
-     * @param {string[]} keys
-     */
-    incrementSortState: function(columnIndex, keys) {
-        var sorts = this.getSortedColumnIndexes(),
-            sortSpec = sorts.find(function(spec, index) {
-                return spec.columnIndex === columnIndex;
-            });
-
-        if (!sortSpec) { // was unsorted
-            if (keys.indexOf('CTRL') < 0) { sorts.length = 0; }
-            sorts.unshift({
-                columnIndex: columnIndex, // so define and...
-                direction: 1 // ...make ascending
-            });
-        } else if (sortSpec.direction > 0) { // was ascending
-            sortSpec.direction = -1; // so make descending
-        } else { // was descending
-            this.unSortColumn(columnIndex, true); // so make unsorted
-        }
-
-        //Minor improvement, but this check can happe n earlier and terminate earlier
-        if (sorts.length > 3) {
-            sorts.length = 3;
-        }
-    },
-
-    /**
-     * @memberOf dataModels.JSON.prototype
      * @param index
      * @param returnAsString
+     * @desc Provides the unicode character used to denote visually if a column is a sorted state
      * @returns {*}
      */
     getSortImageForColumn: function(columnIndex) {
```

Comment:
All I was saying was if we're doing this mixin thing, let's not leave any random "sorting" methods in the core.